docs: remote files can be in lib or routes dir (#14091)
sveltejs/kit: documentation/docs/20-core-concepts/60-remote-functions.md

## Overview

> Remote functions are exported from a `.remote.js` or `.remote.ts` file, and come in four flavours: `query`, `form`, `command` and `prerender`. On the client, the exported functions are transformed to `fetch` wrappers that invoke their counterparts on the server via a generated HTTP endpoint. You can put these remote files in the `lib` or `routes` directory.

Reviewer: ```suggestion
Remote functions are exported from a `.remote.js` or `.remote.ts` file, and come in four flavours: `query`, `form`, `command` and `prerender`. On the client, the exported functions are transformed to `fetch` wrappers that invoke their counterparts on the server via a generated HTTP endpoint. Remote files must be placed in the `lib` or `routes` directory.
```
